content/doc/developer/security/index.adoc

```diff
@@ -96,6 +98,63 @@ To do this, write Jelly like this:
 NOTE: This is not to be confused with the +checkPermission+ invocation in your operation.
 Users can still hit the URL directly, so you still need to protect the operation itself, in addition to disabling the UI rendering
 
+=== Authentication methods
+
+In Jenkins the security engine that is used is AcegiSecurity (ancestor of SpringSecurity).
```

Comment:
Are these intended to be class names? Artifact names? Etc.? If so, wrap in `+`. If not, spell out as given in project pages (`ACEGI Security` IIRC), etc.

Reply:
Corrected as Acegi Security and Spring Security. Source: https://spring.io/blog/2007/01/25/why-the-name-acegi